activiti-core/activiti-engine/pom.xml

```diff
@@ -154,6 +154,10 @@
       <artifactId>hibernate-core</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
```

Comment:
why add mysql connector here? You should add that in your application's pom.xml if you're using mysql.